test/powershell/Modules/Microsoft.PowerShell.Utility/ConvertTo-Json.Tests.ps1

```diff
@@ -22,13 +26,13 @@ Describe 'ConvertTo-Json' -tags "CI" {
         $null = $ps.AddScript({
             $obj = [PSCustomObject]@{P1 = ''; P2 = ''; P3 = ''; P4 = ''; P5 = ''; P6 = ''}
             $obj.P1 = $obj.P2 = $obj.P3 = $obj.P4 = $obj.P5 = $obj.P6 = $obj
-            1..100 | Foreach-Object { $obj } | ConvertTo-Json -Depth 10 -Verbose
+            1..100 | Foreach-Object { $obj } | ConvertTo-Json -Depth 10
             # the conversion is expected to take some time, this throw is in case it doesn't
             throw "Should not have thrown exception"
         })
         $null = $ps.BeginInvoke()
-        # wait for verbose message from ConvertTo-Json to ensure cmdlet is processing
-        Wait-UntilTrue { $ps.Streams.Verbose.Count -gt 0 }
+        # wait that ConvertTo-Json cmdlet is processing
+        Start-Sleep -Milliseconds 300
```

Comment:
Is there a reason for this change from using the verbose channel to know that the cmdlet has started to a sleep?  On a particularly slow container/VM, it might take longer than 300ms.

Reply:
We removed the verbose message in #7487  and currently the test take 10 seconds until timeout.
We could consider adding debug message insted of the verbose message.

Reply:
@iSazonov ok, got it.  Seems like the right thing to do would be to have some test hook so that we can make this predictable as anything with small timeouts are more likely to fail in CI especially when it's under load.

Reply:
I added verbose in `BeginProcessing()`. (Previously it was in `ProcessRecord()`)